.npmignore

```diff
@@ -3,6 +3,7 @@
 !packages/*/lib/**/*
 !packages/*/src/**/*
 !packages/*/index.js
+!packages/datadog-instrumentations/orchestrion.yml
```

Comment:
Nit, but since it's just a config file could we just move it to the root?

Reply:
Since it's effectively a list of definitions of instrumentations, it made sense to me to keep it close to existing instrumentations. I'd prefer that it's in here.